README.md

```diff
@@ -117,3 +117,43 @@ analyzer:
   exclude:
     - 'bazel-*/**'
 ```
+
+### Customizing your generated BUILD files
+
+It is fairly common for a package to want to split up their sources into
```

Comment:
might want to explicitly call out that this section is not necessary if there is only 1 target that includes all sources. It's implied here, but some folks might still assume they need this file/section